spec/koans.js

```diff
@@ -83,56 +96,59 @@ describe('`const` is like `let` plus read-only. ', () => {
 
 describe('`string.includes()` finds string within another string. ', () => {
 
-  describe('find a single character', function() {
-    it('in a three char string', function() {
-      /* const searchString = ???? */
-      //expect('xyz'.includes(searchString)).toBe(true);
+  describe('find a single character', function () {
+    it('in a three char string', function () {
+
+
+      const searchString = 'y'
+      expect('xyz'.includes(searchString)).toBe(true);
     });
-    it('reports false if character was not found', function() {
-      /* const expected = ????*/;
-      //expect('xyz'.includes('abc')).toBe(expected);
+    it('reports false if character was not found', function () {
+      const expected = false;
+      expect('xyz'.includes('abc')).toBe(expected);
     });
   });
 
-  describe('find a string', function() {
-    it('that matches exactly', function() {
-      /* const findSome = .... => 'xyz'.includes();*/
-      //expect(findSome('xyz')).toBe(true);
+  describe('find a string', function () {
+    it('that matches exactly', function () {
+      const findSome = () => 'xyz'.includes('xyz');
+      expect(findSome('xyz')).toBe(true);
     });
   });
 
-  describe('search for an empty string, is always true', function() {
-    it('in an empty string', function() {
-      /* .... */
-      //expect(''.includes(x)).toBe(true);
+  describe('search for an empty string, is always true', function () {
+    it('in an empty string', function () {
+      x = '';
+
+      expect(''.includes(x)).toBe(true);
     });
-    it('in `abc`', function() {
-      /* .... */
-      //expect('abc'.includes(x)).toBe(true);
+    it('in `abc`', function () {
+      x = ''
+      expect('abc'.includes(x)).toBe(true);
     });
   });
 
-  describe('takes a position from where to start searching', function() {
-    it('does not find `a` after position 1 in `abc`', function() {
-      /*....*/
-      //expect('abc'.includes('a', position)).toBe(false);
+  describe('takes a position from where to start searching', function () {
+    it('does not find `a` after position 1 in `abc`', function () {
+      let position = 1;
+      expect('abc'.includes('a', position)).toBe(false);
     });
-    it('even the position gets coerced', function() {
-      /*const findAtPosition = (pos) => 'xyz'.includes(?????);*/
-      //expect(findAtPosition('2')).toBe(true);
+    it('even the position gets coerced', function () {
+      const findAtPosition = (pos) => 'xyz'.includes('z');
+      expect(findAtPosition('2')).toBe(true);
     });
```

Comment:
Aquí en realidad no estáis usando la posición para la búsqueda.

![image](https://user-images.githubusercontent.com/26923217/73739459-869de900-4746-11ea-9d2e-638e49b63d42.png)